src/style.css

```diff
@@ -1 +1,120 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 900;
+}
+
+body {
+  font-weight: 300;
+}
+
+.search-big {
+  display: flex;
+  box-sizing: border-box;
+
+  width: 100%;
+  height: 70px;
+
+  margin-block: 20px;
+
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+}
+
+.search-big:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+}
+
+.search-big:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+}
+
+.search-big__button {
+  width: 62px;
+  height: 100%;
+  padding-inline: 0;
+  border: none;
+
+  background: transparent url(./images/Search.svg) no-repeat 26px center / 19px;
+
+}
+
+.search-big__text {
+  font-family: Avenir, sans-serif;
```

Comment:
Move this property to body selector and here only inherit it

Reply:
I tried it before. It doesn't work so I used selector "*".